vllm/v1/core/sched/scheduler.py

```diff
@@ -152,6 +152,10 @@ def schedule(self) -> SchedulerOutput:
 
             num_new_tokens = (request.num_tokens_with_spec -
                               request.num_computed_tokens)
+            if self.scheduler_config.long_prefill_token_threshold > 0:
+                num_new_tokens = min(
+                    num_new_tokens,
+                    self.scheduler_config.long_prefill_token_threshold)
```

Comment:
Just looking at this now.. small nit that this could be simplified slightly
```python
            if 0 < self.scheduler_config.long_prefill_token_threshold < num_new_tokens:
                num_new_tokens = self.scheduler_config.long_prefill_token_threshold
```

Reply:
Good point. Fixed in #15307